### PR TITLE
Feature/aos 2498 store deployresult elsewhere

### PR DESCRIPTION
**Pull request by bjartek**
Store deploy results in a seperate git repo using the rest api

### CODE REVIEW

**Comment by bjartek** [comment]
DeployLogService is not tested, it was not tested before this PR and it is not tested now. Is this the time to do it or do we want to just take that debt further?

**Comment by bjartek** [comment]
I have manually tested using the http methods that DeployLogService works.

File: src/main/kotlin/no/skatteetaten/aurora/boober/service/BitbucketDeploymentTagService.kt [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+import org.springframework.http.HttpHeaders
+import org.springframework.http.HttpMethod
+import org.springframework.http.MediaType
+import org.springframework.stereotype.Service
+import org.springframework.util.LinkedMultiValueMap
+import org.springframework.util.MultiValueMap
+import org.springframework.web.client.RestTemplate
+
+@Service
+class BitbucketDeploymentTagService(
```
Comment (oyvinder) [inline src/main/kotlin/no/skatteetaten/aurora/boober/service/BitbucketDeploymentTagService.kt]: Should this service be renamed to BitbucketFileService?

**Comment by bjartek** [comment]
AOS-2759 is created to test DeployLogService. It is important to get this deployed.

**Pull request merged**